### PR TITLE
Better docs for gitlab token

### PR DESCRIPTION
**Pull request by eranelbaz**
### Issue & Steps to Reproduce / Feature Request
[//]: <> (choose “resolves” “fixes” or “closes” and put link for the issue)

Since the token id is hidden from the user in our FE we need to add docs about how the user can get this token

### CODE REVIEW

File: docs/resources/template.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ resource "env0_template_project_assignment" "assignment" {
 - **revision** (String) source code revision (branch / tag) to use
 - **ssh_keys** (List of Map of String) an array of references to 'data_ssh_key' to use when accessing git over ssh
 - **terraform_version** (String) Terraform version to use
-- **token_id** (String) The token id used for private git repos or for integration with GitLab
+- **token_id** (String) The token id used for private git repos or for integration with GitLab, you can get it by using data resource on existing template or contact us
```
Comment (omry-hay) [inline docs/resources/template.md:60]:
```suggestion
- **token_id** (String) The token id used for private git repos or for integration with GitLab, you can get it by using data resource of an existing template or contact our support team
```

File: env0/resource_template.go [hunk 2]
```diff
@@ -101,7 +101,7 @@ func resourceTemplate() *schema.Resource {
 			},
 			"token_id": {
 				Type:        schema.TypeString,
-				Description: "The token id used for private git repos or for integration with GitLab",
+				Description: "The token id used for private git repos or for integration with GitLab, you can get it by using data resource on existing template or contact us",
```
Comment (omry-hay) [inline env0/resource_template.go:104]:
```suggestion
				Description: "The token id used for private git repos or for integration with GitLab, you can get this value by using data resource of an existing Gitlab template or contact our support team",
```

**Review by omry-hay — COMMENTED** [review]
Small comments

**Comment by eranelbaz** [comment]
[Better docs for gitlab token](https://trello.com/c/v1ZJVmCo/3946-better-docs-for-gitlab-token)

**Comment by eranelbaz** [comment]
[Better docs for gitlab token](https://trello.com/c/v1ZJVmCo/3946-better-docs-for-gitlab-token)

File: docs/resources/template.md [hunk 3]
```diff
@@ -57,7 +57,7 @@ resource "env0_template_project_assignment" "assignment" {
 - **revision** (String) source code revision (branch / tag) to use
 - **ssh_keys** (List of Map of String) an array of references to 'data_ssh_key' to use when accessing git over ssh
 - **terraform_version** (String) Terraform version to use
-- **token_id** (String) The token id used for private git repos or for integration with GitLab
+- **token_id** (String) The token id used for private git repos or for integration with GitLab, you can get this value by using data resource of an existing Gitlab template or contact our support team
```
Comment (omry-hay) [inline docs/resources/template.md:60]:
```suggestion
- **token_id** (String) The token id used for private git repos or for integration with GitLab, you can get this value by using a data resource of an existing Gitlab template or contact our support team
```

File: env0/resource_template.go [hunk 4]
```diff
@@ -101,7 +101,7 @@ func resourceTemplate() *schema.Resource {
 			},
 			"token_id": {
 				Type:        schema.TypeString,
-				Description: "The token id used for private git repos or for integration with GitLab",
+				Description: "The token id used for private git repos or for integration with GitLab, you can get this value by using data resource of an existing Gitlab template or contact our support team",
```
Comment (omry-hay) [inline env0/resource_template.go:104]:
```suggestion
				Description: "The token id used for private git repos or for integration with GitLab, you can get this value by using a data resource of an existing Gitlab template or contact our support team",
```

**Review by omry-hay — APPROVED** [review]
Small comment, but after that you can merge.

**Pull request merged**